src/main/resources/static/gogo_components/pokemon-view/pokemon-view.html

```diff
@@ -14,7 +14,8 @@
             <div class="thumbnail">
                 <img src="https://img.pokemondb.net/artwork/{{pokemon.pokedexLink}}.jpg" style="height: 200px;"/>
                 <div class="caption">
-                    <h3>{{pokemon.name}}</h3>
+                    <h3>{{pokemon.nickname}}</h3>
```

Comment:
Actually after seeing the screenshot, maybe we could do as suggested here (so display both the nickname and name) only if nickname is different than name, but display the name next to the nickname. With the same formatting as in the screenshot. (italic, smaller font).